{"name": "Проверка ошибки создания курьера: создание двух одинаковых курьеров", "status": "passed", "start": 1699616036545, "stop": 1699616036946, "uuid": "f84ab84c-6319-4f49-8c13-d4ff9157d847", "historyId": "43e43f114b6fcd00132e12b3d5086de8", "testCaseId": "43e43f114b6fcd00132e12b3d5086de8", "fullName": "tests.test_courier_creation.TestCourierCreation#test_courier_duplicate_courier_failed_create", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_creation"}, {"name": "subSuite", "value": "TestCourierCreation"}, {"name": "host", "value": "WKS-084"}, {"name": "thread", "value": "20968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_creation"}]}